src/olympia/reviewers/models.py

```diff
@@ -1023,10 +1025,25 @@ def calculate_weight(self):
         The weight value is then used in reviewer tools to prioritize add-ons
         in the auto-approved queue, the weight_info shown to reviewers in the
         review page."""
+        non_code_weight_factors = self.calculate_non_code_weight_factors()
+        code_weight_factors = self.calculate_static_analysis_weight_factors()
```

Comment:
maybe we should aim for consistency here and therefore use `code_weight` instead of `static_analysis`